Dockerfile

```diff
@@ -1,42 +1,29 @@
-# Copyright 2020 NVIDIA
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-FROM golang:alpine as builder
-
-COPY . /usr/src/network-operator
-
-ENV HTTP_PROXY $http_proxy
-ENV HTTPS_PROXY $https_proxy
-
-WORKDIR /usr/src/network-operator
-RUN apk add --no-cache --virtual build-dependencies build-base linux-headers git && \
-    make clean && \
-    make build
-
-FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
-
-ENV OPERATOR=/usr/local/bin/network-operator \
-    USER_UID=1001 \
-    USER_NAME=network-operator
-
-# Copy manifest dir
-COPY --from=builder /usr/src/network-operator/manifests /etc/manifests
-
-# install operator binary
-COPY --from=builder /usr/src/network-operator/build/_output/network-operator ${OPERATOR}
-COPY --from=builder /usr/src/network-operator/build/bin /usr/local/bin
-LABEL io.k8s.display-name="Mellanox Network Operator"
-RUN  /usr/local/bin/user_setup
-
-ENTRYPOINT ["/usr/local/bin/entrypoint"]
-USER ${USER_UID}
+# Build the manager binary
+FROM golang:1.16 as builder
+
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the go source
+COPY main.go main.go
+COPY api/ api/
+COPY controllers/ controllers/
+COPY pkg/ pkg/
+
+# Build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
+
+# Use distroless as minimal base image to package the manager binary
+# Refer to https://github.com/GoogleContainerTools/distroless for more details
+FROM gcr.io/distroless/static:nonroot
+WORKDIR /
+COPY --from=builder /workspace/manager .
+COPY manifests/ manifests/
```

Comment:
original destination for `manifests` was `/etc/manifests` will this work ?
https://github.com/Mellanox/network-operator/blob/e73e49b99ff9d2f0eb92ab95143a9b4890f96d07/deploy/operator.yaml#L44

Reply:
Yep, works